### DISCUSSION TITLE
Cannot connect to the database

### DISCUSSION THREAD

**Discussion by sciencekiller** [nextcloud/all-in-one#4647] [Questions] [answered]
I have deployed the nextcloud aio on my server, after I click the deploy button, I get the exception " psql: error: connection to server at "nextcloud-aio-database" (172.20.0.4), port 5432 failed: FATAL:  password authentication failed for user "oc_nextcloud" "

**Comment by NeuroTerm** [comment]
nextcloud-aio-database as database host, oc_nextcloud as database username and the password that you get when running sudo docker exec nextcloud-aio-nextcloud grep dbpassword config/config.php as the password.

**Comment by szaimen** [comment, marked as answer]
Hi, this looks like a hiccup. Can you follow https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-properly-reset-the-instance
